mappings/net/minecraft/item/ItemUsageContext.mapping

```diff
@@ -23,4 +23,4 @@ CLASS net/minecraft/class_1838 net/minecraft/item/ItemUsageContext
 	METHOD method_8042 getPlayerFacing ()Lnet/minecraft/class_2350;
 	METHOD method_8044 getPlayerYaw ()F
 	METHOD method_8045 getWorld ()Lnet/minecraft/class_1937;
-	METHOD method_8046 isPlayerSneaking ()Z
+	METHOD method_8046 shouldCancelInteraction ()Z
```

Comment:
:thinking: how the heck did this got wrong

Reply:
It is only true when the player is sneaking, however I decided to rename it based on its usage rather than its implementation. Its implementation even calls `player.shouldCancelInteration`, rather than `player.isSneaking`.

Reply:
Is there any minecraft wiki articles or such that states how sneaking is intended to cancel interactions

Reply:
I don't know, but it does.